src/index.html

```diff
@@ -7,6 +7,26 @@
   <title>Document</title>
 </head>
 <body>
-  <h1>RED</h1>
+  <h1>Leaderboard</h1>
+
+  <section id="scores-section">
+
+   <div id="scores-header">
+    <h2>Recent scores</h2>
+    <button type="button">Refresh</button>
+   </div>
+
+   <div id="scores"></div>
+
+  </section>
+
+  <section id="add-score-section">
+    <h2>Add your score</h2>
+    <form action="#">
+      <input type="text" name="name" id="name" placeholder="Your name">
+      <input type="text" name="score" id="score" placeholder="Your score">
+      <button type="submit" id="submit">Submit</button>
+    </form>
+  </section>
 </body>
```

Comment:
- [ ] While styling is not a requirement, your app layout is expected to match the wireframe. Kindly do that.

  Your layout | Wireframe
  :---:|:---:
  ![alej-main](https://user-images.githubusercontent.com/36542986/156228856-366fdf6b-06d0-4d70-a7bc-80921f138d57.jpg) | ![alej-wire](https://user-images.githubusercontent.com/36542986/156228859-331626e2-5f94-4816-9321-5e96c88992ab.jpg)